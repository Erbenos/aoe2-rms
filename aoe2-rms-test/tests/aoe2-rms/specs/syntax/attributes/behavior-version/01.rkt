#lang aoe2-rms
; Activate new land generation behavior in your script and observe that your lands get smaller.

(behavior-version 1)

<LAND-GENERATION>
(create-player-lands
 (terrain-type 'DLC_BLACK)
 (number-of-tiles 250)
 (base-size 12))
