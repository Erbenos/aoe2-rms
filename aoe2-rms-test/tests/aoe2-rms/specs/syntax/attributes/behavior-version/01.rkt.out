behavior_version 1
<LAND_GENERATION>
create_player_lands {
  terrain_type DLC_BLACK
  number_of_tiles 250
  base_size 12
}
